src/pages/Error/Error.tsx

```diff
@@ -0,0 +1,26 @@
+import "./Error.scss";
+import ErrorImage from "../../assets/images/404Error.png";
```

Comment:
This is not a component so should not be PascalCased the file should be lowercase hyphenated as well. 

`import errorImage from "../../assets/images/404-error.png";`